addon/index.js

```diff
@@ -346,9 +346,13 @@ function updateExperiments() {
 function syncAllAddonInstallations(serverInstalled) {
   const availableIDs = Object.keys(store.availableExperiments);
   const clientIDs = Object.keys(store.installedAddons);
-  const serverIDs = serverInstalled
-    .filter(item => item.client_id === store.clientUUID)
-    .map(item => item.addon_id);
+  const serverIDs = [];
```

Comment:
I'd guess 24 to 48 hours for addon updates, assuming everyone's Firefoxes are running at some point during that period to wake up & check